src/mpl/include/mpl_gpu.h

```diff
@@ -42,7 +38,10 @@ int MPL_gpu_unregister_host(const void *ptr);
 int MPL_gpu_malloc(void **ptr, size_t size, MPL_gpu_device_handle_t h_device);
 int MPL_gpu_free(void *ptr);
 
-int MPL_gpu_init(void);
+int MPL_gpu_init(int local_size, int local_rank, int need_thread_safety);
 int MPL_gpu_finalize(void);
 
+int MPL_gpu_ipc_handle_create(const void *addr, MPL_gpu_ipc_mem_handle_t * ipc_handle);
+int MPL_gpu_ipc_handle_map(MPL_gpu_ipc_mem_handle_t ipc_handle, void **addr);
```

Comment:
This basically looks like a rename of the variable type from `ptr` to `addr` and moving of the functions.  I'd recommend reverting this part.  All other functions seem to use `ptr`, so it's better to be consistent.